internal/store/nats.go

```diff
@@ -174,9 +175,58 @@ func (n *natsStore) GetServer(ctx context.Context, serverID uuid.UUID) (*model.S
 	return &model.Server{ID: obj.UUID, FacilityCode: obj.FacilityCode}, nil
 }
 
-// List is deprecated and is not implemented here
-func (n *natsStore) List(_ context.Context, _ uuid.UUID, _ rctypes.State) ([]*rctypes.Condition, error) {
-	return nil, errors.New("unimplemented")
+// List retrieves any active conditions
+func (n *natsStore) List(ctx context.Context, srvID uuid.UUID, incState rctypes.State) ([]*rctypes.Condition, error) {
```

Comment:
should we call this method `ListActive()` so its role is clearer upfront?
And since the method only returns the first found active condition, I wonder if it should be called something else altogether.

Reply:
Right now, this function needs to implement the existing Repository [interface](https://github.com/metal-toolbox/conditionorc/blob/42b392243ba12cd425c3583e59d92978e0554f35/internal/store/interface.go#L24)